deterministic-build.yml

```diff
@@ -0,0 +1,33 @@
+trigger:
+- master
+
+strategy:
+  matrix:
+    lin:
+      runtime: 'linux-x64'
+    mac:
+      runtime: 'osx-x64'
+    win:
+      runtime: 'win7-x64'
+
+pool:
+  vmImage: ubuntu-16.04
+
+steps:
+- task: UseDotNet@2
+  displayName: 'Install .NET Core 3'
+  inputs:
+    version: 3.1.x
+
+- task: Bash@3
+  displayName: 'Build and compare'
+  inputs:
+    targetType: 'inline'
+    script: |
```

Comment:
What does this pipe denotes?

Reply:
Is the script return return code 0 or something like that? Or how do you signal success vs error?

Reply:
Right, just making sure.

Reply:
The pipe was added by azure pipeline editor. Can it cause a problem?

Reply:
I don't think it can cause a problem. It's more like, I don't know what it exactly does.